src/mctpd.c

```diff
@@ -1832,6 +2021,18 @@ static int method_learn_endpoint(sd_bus_message *call, void *data, sd_bus_error
 	return rc;
 }
 
+static int method_setup_endpoint_by_config(sd_bus_message *call, void *data, sd_bus_error *berr)
+{
+	int rc = 0;
+	ctx *ctx = data;
+	rc = setup_static_eid(ctx);
```

Comment:
no handling here? if not this is what you intended, then drop the assignment.

Reply:
Done